operator/internal/manifests/internal/config/loki-config.yaml

```diff
@@ -233,19 +232,21 @@ limits_config:
     enabled: true
     desired_rate: {{ . }}MB
 {{- end }}
-{{- with .Stack.Limits.Global.OTLP }}
+{{- with .OTLPAttributes.Global }}
   otlp_config:
-    {{- with .ResourceAttributes }}
+    {{- if .ResourceAttributes }}
     resource_attributes:
-      ignore_defaults: {{ .IgnoreDefaults }}
-      {{- with .Attributes}}
+      {{- if .IgnoreGlobalStreamLabels }}
+      ignore_defaults: true
+      {{- end }}
```

Comment:
IIUC this will never happen since we never set this field for the `.OTLPAttributes.Global`